lib/typo_killer/finder.ex

```diff
@@ -12,8 +12,10 @@ defmodule TypoKiller.Finder do
 
   def find_typos({words, dictionary}) do
     words
-    |> Enum.map(&Task.async(fn -> calculate_distance(&1, dictionary) end))
-    |> Enum.flat_map(&Task.await(&1, :infinity))
+    |> Flow.from_enumerable()
+    |> Flow.map(fn word -> calculate_distance(word, dictionary) end)
+    |> Flow.reduce(fn -> MapSet.new() end, fn a, b -> MapSet.union(MapSet.new(a), b) end)
```

Comment:
```suggestion
    |> Flow.reduce(fn -> MapSet.new() end, fn a, acc -> MapSet.union(MapSet.new(a), acc) end)
```

I think something more meaningful for `a` and `b` here would be nice :thinking:

Reply:
(not that x is better, but I lack creativity right now :laughing:)

Reply:
maybe `word` and `words_set`?